Introduction to Kotlin Multiplatform/07_js_canvas.md

```diff
@@ -1,13 +1,15 @@
-# HTML Canvas for Rendering
+# Common Rendering and JS
 
-Now we are ready to update the JS code to implement the `expect`-ed declarations. 
+Now we are ready to update our Kotlin/JS code to provide 
+`actual` declarations for all the `expect` declarations from the `commonMain`. 
 We will be using the [HTML Canvas](https://www.w3schools.com/html/html5_canvas.asp)
 on the client-side to render the images.
 
 ## Adding Colors
 
 The actual `Color` class for the client-side can be implemented from scratch.
```

Comment:
The actual `Color` class for the client-side can be implemented from scratch.
Let's create a `colour-actual.kt` file in the `src/jsMain/kotlin` folder
with the following contents